config/jobs/kubernetes-sigs/testing_frameworks/testing-frameworks-config.yaml

```diff
@@ -5,15 +5,32 @@ presubmits:
     - master
     always_run: true
     skip_report: false
+    decorate: true
     labels:
       preset-service-account: "true"
+      preset-dind-enabled: "true"
```

Comment:
This is essentially a macro to inject an environment variable, volume and volumeMount into your pod (and adding these three items into your pod directly will have the same effect for better or worse).

https://github.com/kubernetes/test-infra/blob/ee541c6392a83f64a1ccd7430de0386c75a1c7e7/prow/config.yaml#L556-L566

Reply:
Oh there they are! Thanks for pointing me to the preset definition!